specs/badge.md

```diff
@@ -0,0 +1,71 @@
+# fast-badge
+
+## Overview
+
+*Badge* component is used to highlight an item and attract attention or flag status, such as “New”, “Sale”, or other short phrases or a number to represent unread messages, updates available, etc..
+
+### Use Cases
+
+Typical use cases include, but are not limited to, denoting a sale or new item, flagging an item as part of a category or representing a value of unread messages.
+  
+### Features
+
+A badge should allow the following properties:
+- `fill`, a string that passes the color value for the backplate. Defaults to "transparent".
+- `size`, the size the badge is rendered.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/badge)
+- [Material UI](https://material-ui.com/components/badges/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/badges/)
+- [Ant Design Badge](https://ant.design/components/badge/)
+- [Ant Design Lozenge](https://atlaskit.atlassian.com/packages/core/lozenge)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/badge)
+
+---
+
+### API
+
+*Component name:*
+- `fast-badge`
+
+*Attributes:*
+- `fill`: string
+- `size`: enum
+    - `small`
+    - `large`
+
+*Slots:*
+- `default`
+
+### Anatomy and Appearance
+
+*Template:*
+```
+<span style={{background: #FFD800}}>
+    <slot></slot>
+</span>
+```
+
+## Implementation
+
+```
+<fast-badge
+    fill="#FFD800"
+    size="large"
+>
+    New
+</fast-badge>
+```
+
+### Accessibility
+
+Ensure text meets WCAG 2.1 color contrast against background.
```

Comment:
This is a good callout that I didn't see prior. Have you thought about how we might support this from an API standpoint? If I consider that I can set any particular value as the background of the badge, how do we support or ensure this?

Reply:
I presumption is we will use whatever utility @EisenbergEffect comes up with that fills the role that our `<Background/>` react component did. So whatever fill value is passed the foreground color `--neutral-foreground-rest` will adjust to ensure proper contrast.